osgtest/tests/test_55_condorce.py

```diff
@@ -137,23 +138,33 @@ def test_06_use_gums_auth(self):
                       '# globus_mapping liblcas_lcmaps_gt4_mapping.so lcmaps_callout',
                       'globus_mapping liblcas_lcmaps_gt4_mapping.so lcmaps_callout',
                       owner='condor-ce')
-
-        service.check_stop('condor-ce')
-        service.check_start('condor-ce')
         core.state['condor-ce.gums-auth'] = True
 
-    def test_07_ping_with_gums(self):
-        self.general_requirements()
-        core.skip_ok_unless_installed('gums-service')
-        self.skip_bad_unless(core.state['condor-ce.gums-auth'], 'CE not using GUMS auth')
-
-        # Wait for the schedd to come back up
+        service.check_stop('condor-ce')
         try:
             stat = os.stat(core.config['condor-ce.collectorlog'])
         except OSError:
             stat = None
+
+        service.check_start('condor-ce')
+        # Wait for the schedd to come back up
         self.failUnless(condor.wait_for_daemon(core.config['condor-ce.collectorlog'], stat, 'Schedd', 300.0),
                         'Schedd failed to restart within the 1 min window')
         command = ('condor_ce_ping', 'WRITE', '-verbose')
         stdout, _, _ = core.check_system(command, 'ping using GSI and gridmap', user=True)
         self.assert_(re.search(r'Authorized:\s*TRUE', stdout), 'could not authorize with GSI')
+
+    def test_07_ceview(self):
+        core.config['condor-ce.view-listening'] = False
+        self.general_requirements()
+        core.skip_ok_unless_installed('htcondor-ce-view')
+        view_sock = socket.socket()
+        try:
+            try:
+                view_sock.connect((core.get_hostname(), int(core.config['condor-ce.view-port'])))
```

Comment:
I think my initial reasoning was to just do the absolute basic check to see that the CE View was listening on the expected port. Using `urllib` seems reasonable and we can do a basic assertion of the contents to ensure that it is indeed the CE VIew that's listening on the port. I'll make a quick commit and test it.